app/Actions/Album/Create.php

```diff
@@ -20,8 +17,11 @@ public function create(string $title, int $parent_id): Album
 		$album = $this->albumFactory->makeFromTitle($title);
 
 		$this->set_parent($album, $parent_id);
+		if (!$album->save()) {
+			throw new \RuntimeException('could not persist album tot DB');
```

Comment:
Why not create a proper exception inheriting from RuntimeException ?

Reply:
> Why not create a proper exception inheriting from `RuntimeException` ?

I also thought of that. However, I wanted to postpone that to another PR. IMHO, the whole error handling needs to be re-considered. At the moment we have some error replies which use `JsonError` and `JsonWarning` as a HTTP response object, sometimes we through exceptions (of different classes) and sometime we return 'false'. I totally agree that this requires a complete clean-up in a consistent way. But I did not wanted to open that can of worms at the moment.